sed/loader/flash/loader.py

```diff
@@ -924,7 +926,7 @@ def read_dataframe(
 
         df, df_timed = self.parquet_handler(data_parquet_dir, **kwds)
 
-        metadata = self.parse_metadata() if collect_metadata else {}
+        metadata = self.parse_metadata(**kwds) if collect_metadata else {}
```

Comment:
Regarding documenting the metadata code in examples, I think it will be done later since scicat is still in test instance.

For **kwds, I don't find a large docstring necessarily useful. For someone going through the API on our docs (or looking at code on IDE), they can see relevant keyword arguments documented for relevant methods.